.swiftlint.yml

```diff
@@ -0,0 +1,5 @@
+
+disabled_rules:
+    - closure_body_length
+    - trailing_whitespace
+line_length: 180
```

Comment:
Do we really want a line length this long?

The idea behind a maximum line length is that you are supposed to be able to fit all code in your window without Xcode doing any automatic _line wrapping_. All lines that are longer than the maximum length should be broken up with manual line breaks. That way, you can keep your window size the same always, and always see the code exactly as it is written. 

Keep in mind that when doing code reviews in GitHub, we need to fit two versions of the code side by side. Common maximum values are 80, 120 or 160. 180 is extremely wide and won't fit for me unless I'm using the ultra wide monitors at the office.

I suspect that you increased this as a workaround for long lines that should instead be broken up. For example, this long line on ContentView:31:

```swift
.sheet(isPresented: $viewModel.showingExplanation, content: {ExplanationSheet(viewModel: ExplanationSheetVM(), num: viewModel.comicObject.num)})
```

This should be broken up to become readable, and will easily then fit within 140:

```swift
.sheet(
    isPresented: $viewModel.showingExplanation,
    content: {
        ExplanationSheet(viewModel: ExplanationSheetVM(), num: viewModel.comicObject.num)
    }
)
```

Or even better, use _trailing closure_:

```swift
.sheet(isPresented: $viewModel.showingExplanation) {
    ExplanationSheet(viewModel: ExplanationSheetVM(), num: viewModel.comicObject.num)
}
```

Reply:
![Skärmavbild 2022-02-23 kl  10 03 48](https://user-images.githubusercontent.com/10262269/155288783-2f89441b-1783-4d7d-a587-96b34a5db0ea.png)